diff -uNr sources.orig/wine/dlls/winecoreaudio.drv/authorization.m sources/wine/dlls/winecoreaudio.drv/authorization.m
--- sources.orig/wine/dlls/winecoreaudio.drv/authorization.m	2021-01-26 04:24:29.000000000 +0900
+++ sources/wine/dlls/winecoreaudio.drv/authorization.m	2021-02-08 18:48:39.000000000 +0900
@@ -26,7 +26,8 @@
 
 #include "coreaudio_cocoa.h"
 
-#ifdef HAVE_AVFOUNDATION_AVFOUNDATION_H
+// #ifdef HAVE_AVFOUNDATION_AVFOUNDATION_H
+#if 0
 
 #if !defined(MAC_OS_X_VERSION_10_14) || MAC_OS_X_VERSION_MAX_ALLOWED < MAC_OS_X_VERSION_10_14
 
diff -uNr sources.orig/wine/dlls/winevulkan/vulkan_private.h sources/wine/dlls/winevulkan/vulkan_private.h
--- sources.orig/wine/dlls/winevulkan/vulkan_private.h	2021-01-26 04:24:30.000000000 +0900
+++ sources/wine/dlls/winevulkan/vulkan_private.h	2021-02-08 19:13:35.000000000 +0900
@@ -122,7 +122,7 @@
 
 static inline struct wine_cmd_pool *wine_cmd_pool_from_handle(VkCommandPool handle)
 {
-    return (struct wine_cmd_pool *)(uintptr_t)handle;
+    return handle;
 }
 
 static inline VkCommandPool wine_cmd_pool_to_handle(struct wine_cmd_pool *cmd_pool)
diff -uNr sources.orig/wine/dlls/winevulkan/winevulkan.spec sources/wine/dlls/winevulkan/winevulkan.spec
--- sources.orig/wine/dlls/winevulkan/winevulkan.spec	2021-01-26 04:24:30.000000000 +0900
+++ sources/wine/dlls/winevulkan/winevulkan.spec	2021-02-08 19:15:30.000000000 +0900
@@ -36,7 +36,7 @@
 
 @ stdcall -private vk_icdGetInstanceProcAddr(ptr str) wine_vk_icdGetInstanceProcAddr
 @ stdcall -private vk_icdNegotiateLoaderICDInterfaceVersion(ptr) wine_vk_icdNegotiateLoaderICDInterfaceVersion
-@ cdecl -norelay native_vkGetInstanceProcAddrWINE(ptr str)
+# @ cdecl -norelay native_vkGetInstanceProcAddrWINE(ptr str)
 @ stdcall -private wine_vkAcquireNextImage2KHR(ptr ptr ptr)
 @ stdcall -private wine_vkAcquireNextImageKHR(ptr int64 int64 int64 int64 ptr)
 @ stdcall -private wine_vkAllocateCommandBuffers(ptr ptr ptr)
